src/components/headfooter/headfooter.js

```diff
@@ -40,7 +40,8 @@ const overwriteTheme = defaultTheme => {
 		palette: newPalette,
 		typography: newTypography,
 		// governs the maxWidth of all content
-		maxWidth: '940px'
+		maxWidth: '940px',
+		backgroundColor: '#352A3A'
```

Comment:
I was not clear about this earlier. But M-UI actually offers us a way to define the main theme color of our webpage. 

Refer to this default theme, https://material-ui.com/customization/default-theme/#explore.

Instead of adding a new field called `backgroundColor`, we can overwrite the `palette` object instead. (you can actually see that I overwrote it in this file in `overwriteTheme` function.) Currently, our `primary` color is purple. You can tell by property name that we want the `#352A3A` color to be in `primary.dark`. 

It is better to stick to the convention so later people can use the same purple without having to know the custom field  `backgroundColor` that we added! The reason why we added the `backgroundColor` field is that there is no such convention within the MUI style object. But for coloring, there are conventions.